frontend/server/src/Controllers/School.php

```diff
@@ -68,7 +69,8 @@ public static function getSchoolProfileDetailsForSmarty(\OmegaUp\Request $r): ar
                 )
             );
             if (!is_null($country)) {
-                $details['country_name'] = $country->name;
+                $details['country']['id'] = strval($country->country_id);
```

Comment:
qué no esto estaría accediendo `null`?

```suggestion
                $details['country'] = [
                    'id' => strval($country->country_id),
                    'name' = strval($country->name),
                ];
```